src/components/FeatureCards/FeatureCards.tsx

```diff
@@ -1,47 +1,67 @@
 import type { Breakpoint } from '@mui/material';
 import { useMediaQuery, useTheme } from '@mui/material';
-import { useEffect, useMemo, useState } from 'react';
+import { useMemo } from 'react';
 import { FeatureCard } from 'src/components';
 import { useFeatureCards } from 'src/hooks';
 import { useSettingsStore } from 'src/stores';
-import type { FeatureCardData } from 'src/types';
 import { shallow } from 'zustand/shallow';
 import { FeatureCardsContainer } from '.';
+import { usePersonalizedFeatureCards } from '../../hooks/usePersonalizedFeatureCards';
 
 export const FeatureCards = () => {
-  const [featureCards, setFeatureCards] = useState<FeatureCardData[]>([]);
   const [disabledFeatureCards, welcomeScreenClosed] = useSettingsStore(
     (state) => [state.disabledFeatureCards, state.welcomeScreenClosed],
     shallow,
   );
 
-  const { featureCards: data, isSuccess } = useFeatureCards();
-  const featureCardsFetched = useMemo(() => {
-    if (Array.isArray(data) && !!data.length) {
-      return data?.filter(
-        (el, index) =>
-          isSuccess &&
-          el.attributes.DisplayConditions &&
-          !disabledFeatureCards.includes(el.attributes.DisplayConditions?.id),
-      );
+  const { featureCards: cards, isSuccess } = useFeatureCards();
+  const { featureCards: personalizedCards } = usePersonalizedFeatureCards();
+
+  const slicedFeatureCards = useMemo(() => {
+    if (Array.isArray(cards) && !!cards.length) {
+      return cards
+        ?.filter(
+          (el, index) =>
+            isSuccess &&
+            el.attributes.DisplayConditions &&
+            !disabledFeatureCards.includes(el.attributes.DisplayConditions?.id),
+        )
+        .slice(0, 2);
     }
-    // trigger featureCardsFetched-filtering only once
+    //trigger featureCardsFetched-filtering only once
     // eslint-disable-next-line react-hooks/exhaustive-deps
-  }, [data, isSuccess]);
+  }, [cards, isSuccess]);
 
-  useEffect(() => {
-    if (Array.isArray(featureCardsFetched)) {
-      !!featureCardsFetched.length &&
-        setFeatureCards(featureCardsFetched?.slice(0, 4));
+  const slicedPersonalizedFeatureCards = useMemo(() => {
+    if (Array.isArray(personalizedCards) && !!personalizedCards.length) {
+      return personalizedCards
+        ?.filter(
+          (el, index) =>
+            el.attributes.DisplayConditions &&
+            !disabledFeatureCards.includes(el.attributes.DisplayConditions?.id),
+        )
+        .slice(0, 1);
     }
-  }, [featureCardsFetched]);
+    //trigger featureCardsFetched-filtering only once
```

Comment:
Why do we want to trigger this only once?

Reply:
Thomas has applied it from the existing useMemo. Up there, at some point I had added the "eslint-disable-next-line" in order to only run this snippet once and not repeatedly trigger it. To be honest, now I have tried it without and it seems to work as well. Wdyt?

Anyways, it seems as we don´t need it on the _slicedPersonalizedFeatureCards_ because it does not affect the deps. We can get rid of this (line 39+40)
```
    //trigger featureCardsFetched-filtering only once
    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Reply:
Yes, I think we should avoid disabling the hooks rules unless it is really needed and even in that way it shows that we are doing something wrong 😄